docs/notebooks/beta_regression.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>This is obviously wrong, and it's effectively the same issue highlighted above. Is there some mistake in putting the parameter values back in the outcome space here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/442/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>It took me some minutes to figure out what was the problem. And it was that you were calling <code>expit()</code>  several times. It involves the last two cells of code. I only write the updated versions</p><p> </p><pre class="ql-syntax" spellcheck="false">mean_effect = lag_fitted.posterior.batting_avg_shift.values.mean()

hdi = az.hdi(lag_fitted.posterior.batting_avg_shift, hdi_prob=.95)
lower = expit(hdi.batting_avg_shift[0]).item()
upper = expit(hdi.batting_avg_shift[1]).item()
print(f"Mean effect: {expit(mean_effect):.4f}")
print(f"95% interval {lower:.4f} - {upper:.4f}")

intercept = lag_fitted.posterior.Intercept.values.mean()
</pre><p> </p><pre class="ql-syntax" spellcheck="false">x = np.linspace(df_shift.batting_avg_shift.min(), df_shift.batting_avg_shift.max(), 100)
linear = mean_effect * x + intercept
plt.plot(x, expit(linear), c="black")
plt.xlabel("Previous Year's Batting Average")
plt.ylabel("Batting Average")
</pre><p> </p><p>The problem was that <code>mean_effect</code>  was not the slope parameter, but <code>expit(slope)</code>  and the same thing for <code>intercept</code>,  it was using <code>expit(intercept)</code>.  This does <code>expit(intercept + slope * x)</code></p><p> </p><p> </p><p>This is the updated figure</p><p> </p><p><a href="https://imgur.com/RBS58tD" target="_blank">https://imgur.com/RBS58tD</a></p><p> </p><p> </p><p> </p>